LobbyServer2/ApiServer/StatusController.cs

```diff
@@ -121,10 +125,52 @@ public static Player Of(PersistedAccountData acc)
                 {
                     accountId = acc.AccountId,
                     handle = acc.Handle,
-                    bannerBg = acc.AccountComponent.SelectedBackgroundBannerID == -1  ? 95 : acc.AccountComponent.SelectedBackgroundBannerID, // if no Banner is set default to 95
-                    bannerFg = acc.AccountComponent.SelectedForegroundBannerID == -1  ? 65 : acc.AccountComponent.SelectedForegroundBannerID, // if no Foreground Banner is set default to 65
+                    bannerBg = acc.AccountComponent.SelectedBackgroundBannerID == -1 ? 95 : acc.AccountComponent.SelectedBackgroundBannerID, // if no Banner is set default to 95
+                    bannerFg = acc.AccountComponent.SelectedForegroundBannerID == -1 ? 65 : acc.AccountComponent.SelectedForegroundBannerID, // if no Foreground Banner is set default to 65
+                    titleId = acc.AccountComponent.SelectedTitleID,
+                    status = FriendManager.GetStatusString(SessionManager.GetClientConnection(acc.AccountId)),
+                    factionData = TrustWarManager.PlayerTrustWarDetails.Of(acc),
+                };
+            }
+        }
+
+        public struct ActivePlayer
+        {
+            public long accountId { get; set; }
+            public string handle { get; set; }
+            public int bannerBg { get; set; }
+            public int bannerFg { get; set; }
+            public string status { get; set; }
+            public int titleId { get; set; }
```

Comment:
Maybe we could just reuse Player here?

Reply:
Yes however it makes an other object inside {} i want it to be same and not have nested objects, 

just change what you want :P